common/character_interactions/00_alliance.txt

```diff
@@ -97,6 +97,11 @@
 				}
```

Comment:
What about enabling animation once someone is stunned?
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>

Reply:
Can't see image

Reply:
> Can't see image

It doesn't play much role. The fact is stunned characters don't change animation to something more fitting.

Reply:
![bild](https://user-images.githubusercontent.com/4825809/117138229-0469fd80-adab-11eb-8352-4eac5469e669.png)

Reply:
Now uses the `still` animation when stunned, tell me if another would be better

Reply:
Now uses `pain` animation